amy/templates/navigation.html

```diff
@@ -78,11 +78,10 @@
           <li class="nav-item dropdown">
             <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Hello, {{ user.get_short_name }}</a>
             <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-              {% if user.is_admin %}
+
               <a class="dropdown-item" href="{% url 'person_details' user.id %}">Your profile</a>
-              {% else %}
-              <a class="dropdown-item" href="{% url 'autoupdate_profile' %}">Your profile</a>
-              {% endif %}
+              <a class="dropdown-item" href="https://carpentries.github.io/amy/users_guide/" target="_blank">Users Guide</a>
```

Comment:
After `target=...` please add `rel="noreferrer nofollow"`. This is a privacy enhancements, so that github won't know where user comes from once they click this link.